crates/c_api/src/types/import.rs

```diff
@@ -53,18 +54,24 @@ pub extern "C" fn wasm_importtype_new(
 
 /// Returns a shared reference to the module namespace of the [`wasm_importtype_t`].
 #[no_mangle]
+#[cfg_attr(
+    feature = "prefix-symbols",
+    export_name = "wasmi_wasm_importtype_module"
+)]
```

Comment:
This should use the `#[cfg_attr(feature = "prefix-symbols", wasmi_c_api_macros::prefix_symbol)]` attribute. Probably an oversight due to text replacement without multi-lines.